### DISCUSSION TITLE
🚀 Distro 2023.02.0 Released

### DISCUSSION THREAD

**Discussion by Nicogene** [robotology/community#613] [Releases]
# 2023.02.0 Distro Announcement

## Brief description of the announcement

Hello @robotology/everyone  👋

The [iCub project](https://icub.iit.it/) is happy to announce our **Software Distro 2023.02.0** 📦

This is a major release that comprises **binary packages** as well. Therefore, follow the [instructions][1] available in our documentation to upgrade your system and/or download the new dependencies.

You can find the tags composing this distro in our [Software Versioning Table][2].

The apt binaries cover the following platforms:
- Ubuntu 20.04 Focal 64-bits.
- Ubuntu 22.04 Jammy 64-bits.
- Debian 10 Buster 64-bits.

## Detailed context

This distro is shipped with several updates and fixes, including the ones listed hereafter:

#### 🔘 `YARP-3.8.0`

YARP 3.8.0 is now available 🎉
Release notes and download links can be found at:
- https://github.com/robotology/yarp/releases/v3.8.0.

#### 🔘 Support for multi-core processors on our embedded boards

We improved our FW infrastructure to support the development on boards sporting multi-core processors.
- https://github.com/robotology/icub-main/pull/850

#### 🔘 Added `sensorName` parameter on `embObjIMU` device

- https://github.com/robotology/icub-main/pull/853
- https://github.com/robotology/robots-configuration/pull/422

#### 🔘 Fixed `iCubLisboa01` and `R1SN000` to the use of the latest NWS/NWC layers
- https://github.com/robotology/robots-configuration/pull/446

#### 🔘 Updated the PID gains controlling the neck of the ETH robots
- https://github.com/robotology/robots-configuration/pull/447

#### 🔘 `ergocub-software`

2023.02.0 is the first Distro that includes [`ergocub-software`](https://github.com/icub-tech-iit/ergocub-software), the repository that contains [ergoCub](https://ergocub.eu/) specific software. Right now it hosts the urdf and the urdf generation pipeline.

#### 🔘 `walking-controllers-0.7.0`

[ergoCub](https://ergocub.eu/) can walk with the control architecture implemented in walking-controllers.

#### 🔘 `iCub_SIM` has been removed
As announced when we released the [Distro 2022.11.0][distro-2022.11.0], the old simulator`iCub_SIM` has been removed from our ecosystem via:
- https://github.com/robotology/icub-main/pull/844

[distro-2022.11.0]: https://github.com/robotology/community/discussions/607

[1]: https://icub-tech-iit.github.io/documentation/sw_installation/
[2]: https://icub-tech-iit.github.io/documentation/sw_versioning_table/#table

[^1]: See https://github.com/robotology/icub-main/pull/808
[^2]: See https://github.com/robotology/icub-main/issues/842#issuecomment-1327388046

**Comment by traversaro** [comment]
The 2023.02.1 release was tagged as well, with some minor bug-fixes: https://github.com/robotology/robotology-superbuild/releases/tag/v2023.02.1 .

**Comment by Nicogene** [comment]
We release the [Distro 2023.02.2](https://icub-tech-iit.github.io/documentation/sw_versioning_table/2023.02.2/) which contains some important fixes  such as:
- https://github.com/robotology/robots-configuration/issues/405
- https://github.com/robotology/robots-configuration/issues/462
- https://github.com/robotology/icub-firmware/pull/362
- https://github.com/robotology/icub-firmware/pull/361
- https://github.com/robotology/icub-models/pull/196

**Comment by Nicogene** [comment]
Today we released the [Distro 2023.02.3](https://icub-tech-iit.github.io/documentation/sw_versioning_table/2023.02.3/), a patch release that contains minor fixes such as:

- https://github.com/robotology/robots-configuration/issues/477
- `iCubOsaka01` has been renamed to `iCubTokyo01` via https://github.com/robotology/robots-configuration/pull/474.
- `iRonCub-Mk1` and `R1SN000` have been removed (kept in Git history).
